core/ts/Poll.ts

```diff
@@ -395,8 +409,9 @@ class Poll implements IPoll {
    * poll has concluded.
    * @param _pollId The ID of the poll associated with the messages to
    *        process
+   * @returns stringified circuit inputs
    */
-  public processMessages = (_pollId: number): any => {
+  public processMessages = (_pollId: number): string[] => {
```

Comment:
why is this an array of strings? thought it'd be an object

Reply:
type madness of `stringifyBigInts()` 🫠